doc-content/jbpm-docs/src/main/asciidoc/Integration/cdi-deployment-service-con.adoc

```diff
@@ -0,0 +1,44 @@
+[id='cdi-deployment-service-con_{context}']
+
+= The Deployment service for CDI
+
+The `DeploymentService` service  deploys and undeploys deployment units in the runtime environment. When you deploy a unit using this service, the deployment unit becomes ready for execution and a `RuntimeManager` instance is created for it. You can also use the `DeploymentService` to retrieve the following objects:
+
+* The `RuntimeManager` instance for a given deployment ID
+* The `DeployedUnit` instance that represents the complete deployment unit for the given deployment ID
+* The list of all deployed units known to the deployment service
+
+By default, the deployment service stores the deployed units in memory. In the CDI framework, the component that uses the service can save and restore deployment unit information, for example, using a database, file, system, or repository.
```

Comment:
the deployment service does not store anything. It only keeps track of the deployment units that are deployed (in memory)

Reply:
there is a  DeploymentSyncManager in cdi integration that boots up the deployment store and through a DeploymentEventListener stores data in file system.  This stores the state of the deployments in the file system.
in EJB there is EJB for setting up this.

Reply:
found another line with better explanation
https://github.com/mramendi/kie-docs/pull/128/files#diff-0d71c949e46eb4d27d6410120c30e285bff21745355ce07617f067e99bcf42d7R36